package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "dredd",
-  "version": "0.0.0-semantically-released",
+  "name": "@iflix/dredd",
+  "version": "4.3.0-iflix1",
```

Comment:
I understand this was necessary for you to use your fork, but it cannot be accepted to upstream 🙂